pkg/tsdbctl/check.go

```diff
@@ -56,20 +66,46 @@ Arguments:
 - <item-path> (string) [Required] Path to a metric item within the TSDB table.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 
-			if len(args) == 0 {
-				return errors.New("The check command requires an item path.")
+			numArgs := len(args)
+
+			if numArgs == 0 {
+				return errors.New("The check command requires a metric name.")
+			}
+
+			if strings.Contains(args[0], "/") {
+				if len(commandeer.attrs) == 0 {
+					return errors.New("-a (--attrs) is required when specifying a path")
+				}
+				commandeer.objPath = args[0]
+				return commandeer.check()
 			}
 
-			commandeer.objPath = args[0]
+			commandeer.metricName = args[0]
+			commandeer.labels = append(commandeer.labels, utils.Label{Name: "__name__", Value: commandeer.metricName})
+
+			if numArgs > 1 {
+				for _, labelStr := range strings.Split(args[1], ",") {
+					splitLabelParts := strings.SplitN(labelStr, "=", 2)
+					if len(splitLabelParts) < 2 {
+						return errors.Errorf("failed to parse '%s' as a label", labelStr)
+					}
+					commandeer.labels = append(commandeer.labels, utils.Label{Name: splitLabelParts[0], Value: splitLabelParts[1]})
+				}
+			}
 
 			// Initialize parameters
 			return commandeer.check()
 		},
 	}
 
 	cmd.Flags().StringSliceVarP(&commandeer.attrs, "attrs", "a", []string{},
-		"[Required] An array of metric-item blob attribute names, as a\ncomma-separated list. For example: \"_v35\"; \"_v22,_v23\".")
-	cmd.MarkFlagRequired("attrs")
+		"An array of metric-item blob attribute names, as a\ncomma-separated list. For example: \"_v35\"; \"_v22,_v23\".")
+	cmd.Flags().StringVarP(&commandeer.to, "end", "e", "",
+		"End (maximum) time for the query, as a string containing an\nRFC3339 time string, a Unix timestamp in milliseconds, or\na relative time of the format \"now\" or \"now-[0-9]+[mhd]\"\n(where 'm' = minutes, 'h' = hours, and 'd' = days).\nExamples: \"2018-09-26T14:10:20Z\"; \"1537971006000\";\n\"now-3h\"; \"now-7d\". (default \"now\")")
+	cmd.Flags().StringVarP(&commandeer.from, "begin", "b", "",
+		"Start (minimum) time for the query, as a string containing\nan RFC3339 time, a Unix timestamp in milliseconds, a\nrelative time of the format \"now\" or \"now-[0-9]+[mhd]\"\n(where 'm' = minutes, 'h' = hours, and 'd' = days), or 0\nfor the earliest time. Examples: \"2016-01-02T15:34:26Z\";\n\"1451748866\"; \"now-90m\"; \"0\". (default = <end time> - 1h)")
```

Comment:
`RFC3339` -> `RFC 3339`
default value (`now-1`) could be set for the `begin` flag